src/main/java/com/shipmentEvents/demo/createCookie.java

```diff
@@ -0,0 +1,8 @@
+import javax.servlet.http.Cookie;
+
+public static void createCookie() {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on tokens, ClassHeader expected instead